fork.Rmd

```diff
@@ -10,7 +10,7 @@ Date: 2017/06/27
 
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE,
-                      dev = "png")
+                      dev = c("png","svg"))
```

Comment:
Oh! That's really cool, SVG is a really nice, lightweight image format, I think that will be useful!